src/content/docs/new-relic-one/use-new-relic-one/ui-data/maps-in-context.mdx

```diff
@@ -64,29 +66,37 @@ The typical map view shows you entities, represented as hexagons, and their rela
     </tbody>
 </table>
 
-## Health status
+## Health status [#health]
 
-Where relevant, maps add color to an entity to show its alert status. The maps use the standard green, yellow, red, gray palette.
+Some map use our standard [health status colors](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-entity-health-status-find-entities-without-alert-conditions/#colors) to indicate entity health. 
 
-Distributed trace maps fill a primary entity's hexagon with a solid color to reflect health context from trace spans:
+Distributed trace maps use these colors to indicate health:
     
-    * Red hexagons mean one or more span errors.
-    * Orange hexagons mean one or more anomalous spans are detected.
+  * A solid red hexagon means one or more span errors.
+  * A solid orange hexagon means one or more anomalous spans.
 
-## Map tooltip
+## Map tooltip [#tool-tips]
 
-Maps in context show additional information about an entity, external, or datastore when you hover over its hexagon.
+Maps in context show additional information about map elements when you hover over them.
 
 ![A screenshot of a map entity's detail information.](./images/context-map-entity-detail.png "A map entity's detail information.")
 
-<figcaption>When you mouseover an entity on a map, you'll see more information detail.</figcaption>
+<figcaption>When you mouse over an entity on a map, you'll see more detail.</figcaption>
 
 The map tooltip includes:
 
-* the entity's name and type.
-* any essential tags present on the entity.
-* a preview of the entity's golden signals over the last 30 minutes.
-* hexagons representing the hosts this entity is running on, if known.
-* other entities that have manually-created relationships to this entity.
-* a button that opens the entity's mini-overview to gain deeper detail.
-* a button to show all the entity's dependencies.
+* entity name and type
+* any essential [tags](/docs/new-relic-one/use-new-relic-one/core-concepts/use-tags-help-organize-find-your-data) present on the entity
+* golden signals from the last 30 minutes
+* hosts the entity is running on, if known
+* other entities that have manually-created relationships to this entity
+* a link to an entity's mini-overview
+* a link to show all of the entity's dependencies
+
+
+## How map relationships are determined [#relationships]
+
+The feature that the map resides in determines the detected relationships. Examples: 
+
+* Applied Intelligence: when you select an [issue](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/use-incident-intelligence/#issue-summary) in the UI, the map relies on [entity relationships](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial).
```

Comment:
Though you already link to the `Use Incident Intelligence` page in the photo caption, I agree that it could be added again because some people will gloss over the caption. :)